src/components/project/HeaderSection/HeaderSection.tsx

```diff
@@ -0,0 +1,112 @@
+import Image from 'next/image';
+import { css } from '@emotion/react';
+
+import useMediaQuery from '~/hooks/use-media-query';
+import { colors, mediaQuery } from '~/styles/constants';
+import { body1Css } from '~/styles/css';
+
+export default function HeaderSection() {
+  const isMobile = useMediaQuery('xs');
+  return (
+    <div css={sectionCss}>
+      <h1 css={headingCss}>
+        DESIGNERS
+        <br />
+        &PROGRAMMER
+        <br />
+        MEET
+      </h1>
+      <p css={descriptionCss}>
+        독창적이면서도 날카롭고
+        <br />
+        실패를 두려워하지 않는
+        <br />
+        오직 디프만에서만 가능한
+        <br />
+        프로젝트들을 소개합니다.
+      </p>
+      <Image
+        src={'/project/passion_front.webp'}
+        alt=""
+        width={isMobile ? 188 : 700}
+        height={isMobile ? 112 : 416}
+        css={leftImageCss}
+      />
+      <Image
+        src={'/project/growth_front.webp'}
+        alt=""
+        width={isMobile ? 156 : 583}
+        height={isMobile ? 139 : 519}
+        css={rightImageCss}
+      />
```

Comment:
```suggestion
      <Image
        src={'/project/passion_front.webp'}
        alt="디프만"
        width={isMobile ? 188 : 700}
        height={isMobile ? 112 : 416}
        placeholder="blur"
        blurDataURL={'/project/passion_front.webp'}
        priority
        quality={100}
        css={leftImageCss}
      />
```

로드 우선순위랑 퀄리티, placeholder 옵션 사용하면 좋을 거 같아용

그리고 fill 옵션도 사용해서 반응형 대응도 해볼 수 있을 거 같아요 ! (근데 급한 불부터 끄고 나중에 해도 좋아요 ~~) 👍 👍

Reply:
네넵 QA 중에 눈에 띄는 거부터 하구 내일?즈음에 적용해볼게요

Reply:
봐주셔서 감삼다.. 리얼 무지성으로 짯어요